test/test_jit.py

```diff
@@ -992,6 +992,23 @@ def func(a, b):
         b = torch.rand(1, requires_grad=True)
         self.checkScript(func, (a, b), optimize=True)
 
+    @unittest.skipIf(not PY35, "Python 3.5 needed")
+    def test_matmul(self):
+        def func(a, b):
+            return a @ b
```

Comment:
Unfortunately this will still raise a syntax error in Python 3.5, because `skipIf` won't prevent the interpreter form attempting to parse the code. You need to use something similar to how I handled [a test for type annotations](https://github.com/pytorch/pytorch/blob/master/test/test_jit.py#L2141-L2161).